docs/xenit-style-guide/golang.md

```diff
@@ -7,6 +7,72 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Golang (Go) is one of the most common languages at Xenit especially for backend systems and open source projects. It should be the first language choice when starting a new project.
 
+## Program Input
+
+Go has a lot of options when parsing program input flags and commands. On top of the standard `flag` library there are a bunch of other options out there that do the one or the other, or both. For this reason it is good to standardize on a library to use at Xenit. The most popular libraries at this time are [cobra](https://github.com/spf13/cobra) and by extension [pflag](https://github.com/spf13/pflag). While these libraries may be popular they also introduce a lot of opinion and complexity for the project structure.
```

Comment:
Nitpick,

and complexity *for* the project structure.
->
and complexity *to* the project structure.

?